mkdocs.yml

```diff
@@ -12,6 +12,7 @@ nav:
       - Modality specific files:
         - Magnetic Resonance Imaging: 04-modality-specific-files/01-magnetic-resonance-imaging-data.md
         - Magnetoencephalography: 04-modality-specific-files/02-magnetoencephalography.md
+        - Electroencephalography: 04-modality-specific-files/06-electroencephalography.md
```

Comment:
This is confusing. If you want EEG to be third on the list it should be called `03-electroencephalography.md` and other files in this folder renamed.

Reply:
okay, thanks!